frontend/app/components/wp-table/wp-table.directive.html

```diff
@@ -131,7 +131,8 @@
             </accessible-checkbox>
             <span>
               <a class="wp-table--details-link"
-                 ui-sref="work-packages.list.details.overview({workPackageId: row.object.id})">
+                 ui-state="desiredSplitViewState"
+                 ui-state-params="{workPackageId: row.object.id}">
```

Comment:
A one-time binding without extras won't work, as the state will change. This would be something for the `angular-bind-notifier` or something.